### ISSUE TITLE
ACC-283.1 (Level AA) _ Links _Truncation on enabling text spacing

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3262]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Links 
Affected Check Point:	ACC-283.1
Short text:	ACC-283.1 (Level AA) _ Links _Truncation on enabling text spacing

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Go to KubeconfigURL:https://kyma-env-broker.cp.stage.kyma.cloud.sap/kubeconfig/DFEFE4E0-E202-4455-8A61-37511B4D7553
•	Click on  the link https://dashboard.stage.kyma.cloud.sap/clusters and select Connect.
•	Upload the kubeconfing file downloaded in the first step. You can either drag and drop it or select the upload icon to choose it from the device.
•	Next Step
•	Connect to cluster, A new screen get opens
•	

3. Issue Type: Manual
Observed Behavior:
On enabling text spacing option, The ‘Learn More’ links are truncated and not visible completely within the cards under ‘Monitoring and health’ region. 
Screenshot:
 
Screenshot 2: default
 
Expected Behavior:
All the UIs should adapt to applied text spacing settings without loss of information or functionality.

The ability to increase spacing between lines, words, letters, and paragraphs, or any combination of these, may assist a user with effectively reading text.
After adjusting spacing, text should fits into its containing box without overlapping other boxes or content being cut off.

It is recommended that the UI should have a flexible and clean implementation of measurements basing on em/rem and incorporating the line height.
This way it adapts also to the user's space settings per style sheet.
[ACC-283.1 (Level AA) _ Links _Truncation on enabling text spacing.docx](https://github.com/user-attachments/files/16687996/ACC-283.1.Level.AA._.Links._Truncation.on.enabling.text.spacing.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16687997/Issue.processing-09.2021.2.docx)
